src/components/AllMovies/AllMovies.js

```diff
@@ -1,6 +1,6 @@
 import React, { Component } from 'react'
 import Movie from '../Movie/Movie'
-import Search from '../searchBar/searchBar'
+import Search from '../SearchBar/searchBar'
```

Comment:
Why is this happening!?!?!  Going to have to write GH a letter, its driving me crazy!